### ISSUE TITLE
indication of country

### ISSUE THREAD

**Issue by zayn666** [transmission/transmission#1512]
is there any plan to add the name of the country or its flag to the peer list?

**Comment by sT331h0rs3** [comment]
It would be nice to have such feature in my opinion as well.

**Comment by zayn666** [comment]
most torrent clients already have it, id apreciate if someone submited a pull request

**Comment by Draxter** [comment]
This would indeed be a nice feature.

**Comment by Coeur** [comment]
qBittorrent uses https://db-ip.com

**Comment by jcpvdm** [comment]
> #4769 suggests https://www.maxmind.com/en/geoip2-databases

Yes, that link talks about paid geoip2-databases. I should have instead provide this: https://github.com/maxmind
These MaxMind tools facilitate lookup of IP to Country, based on a local GeoIP database. The user would be responsible to setup the database, and then just modify the settings of transmission to point to it. 

MaxMind also provides a lite version of GeoIP databases, which are less accurate but free.
https://dev.maxmind.com/geoip/geolite2-free-geolocation-data?lang=en
